app/main.py

```diff
@@ -2,0 +1,24 @@
+import os.path
+import shutil
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
+    cmd_name = parsed_command[0]
+    src_file = parsed_command[1]
+    dst_file = parsed_command[2]
```

Comment:
Code Style: Use descriptive and correct variable names. `cmd_name`, `src_file`, and `dst_file` are not very descriptive. Consider renaming them to something like `command_name`, `source_file`, and `destination_file`.